flink-kubernetes/src/test/java/org/apache/flink/kubernetes/highavailability/KubernetesStateHandleStoreTest.java

```diff
@@ -1143,4 +1190,31 @@ private TestingLongStateHandleHelper.LongStateHandle addDeletingEntry(
         configMap.getData().put(key, deleting);
         return state;
     }
+
+    private CompletableFuture<Boolean> retryWithFirstFailedK8sOperation(
```

Comment:
```suggestion
    private static CompletableFuture<Boolean> retryWithFirstFailedK8sOperation(
```
We could even make this a static utility method. We're not using any state in there...